jobs/config.json

```diff
@@ -2763,6 +2763,12 @@
     ], 
     "scenario": "kubernetes_e2e"
   }, 
+  "ci-perf-tests-kubemark-100-benchmark": {
+    "args": [
+      "./benchmark/runner.sh"
```

Comment:
Removed dockerized build from that PR as it wasn't serving much useful purpose anyway (https://github.com/kubernetes/perf-tests/pull/61#issuecomment-301935978). Thanks for pointing this.